qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -209,6 +209,15 @@
                 gagent_start_cmd = "qemu-ga -m virtio-serial -p /dev/virtio-ports/org.qemu.guest_agent.0 -F -d"
             isa_serial:
                 gagent_start_cmd = "pgrep qemu-ga || qemu-ga -d -m isa-serial -p /dev/ttyS1 -F"
+        - check_fsinfo:
+            gagent_check_type = fsinfo
+            disk_serial_param = "serial=GAGENT_TEST"
+            blk_extra_params_image1 = ${disk_serial_param}
+            cmd_get_disk = cat /proc/mounts |awk '$2~/^\%s$/{print $1,$3}'
+            Windows:
+                cmd_get_disk = wmic volume where "DriveLetter='%s'" get FileSystem,DeviceID |findstr /v /i FileSystem
+                blk_extra_params_cd1 = ${disk_serial_param}
+                blk_extra_params_virtio = ${disk_serial_param}
```

Comment:
why do not use `blk_extra_params = ***` directly ?  add `serial` to all cdrom and images that use `images_define_by_variables` to generate command.

Reply:
Updated